src/components/Entry.vue

```diff
@@ -313,7 +385,7 @@ export default {
      */
     async authenticate() {
       let latestUserID = parseInt(this.latestEntry["userID"]);
-      await this.authenticateSRN(latestUserID);
+      await this.authenticateID(latestUserID);
```

Comment:
There seem to be a lot of common code between `Entry.vue` and `OTP.vue`, no?

Reply:
Yess the ideal scenario will be to have just one component which will render things based on the JSON